packages/react-styles/src/css/components/Topology/topology-pipelines.css

```diff
@@ -173,6 +208,11 @@
   --pf-topology-pipelines__pill-background--Stroke: var(--pf-topology-pipelines__pill-background--m-warning--Stroke);
 }
 
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-skipped .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--Fill: var(--pf-topology-pipelines__pill-background--m-skipped--Fill);
```

Comment:
Should this be `--pf-topology-pipelines__pill-background--m-selected--m-skipped--Fill`?